packages/core/src/codewhisperer/activation.ts

```diff
@@ -66,6 +66,8 @@ import { updateUserProxyUrl } from './client/agent'
 import { Container } from './service/serviceContainer'
 import { debounceStartSecurityScan } from './commands/startSecurityScan'
 import { securityScanLanguageContext } from './util/securityScanLanguageContext'
+import { registerWebviewErrorHandler } from '../webviews/server'
+import { logAndShowWebviewError } from '../extensionShared'
```

Comment:
can we move the function to a shared util file somewhere? We are basically importing the entire toolkit extension by doing this

Reply:
Fixed in latest commit.